shoop/core/models/_contacts.py

```diff
@@ -170,7 +207,8 @@ def delete(self, *args, **kwargs):
 
     @property
     def groups(self):
```

Comment:
This really should be a top-level method, since now the `groups` is a required property but it is not required per se. 
Example: you add `ThisIsCustomContact(Contact)` and call the `groups` for that, does it have it?